jekyll/faq.md

```diff
@@ -31,30 +42,20 @@ Future versions will additionally include a GC "per thread group"
 and Nim's type system will be enhanced to accurately model this shared
 memory heap.
 
+## How about security and memory safety?
+
+Nim provides memory safety by not performing pointer aritmethic, with
+optional [checks] (https://nim-lang.org/docs/manual.html#pragmas-compilation-option-pragmas),
+traced and untraced references, safe parallelism through a
+[disjoint checker](https://nim-lang.org/docs/manual.html#parallel-spawn-parallel-statement)
+It supports Valgrind through the koch tool, and taint analysis.
```

Comment:
``koch`` is only relevant when debugging the compiler AFAIK

Reply:
``koch valgrind`` exists and prepares valgrind with a Nim specific configuration.

Reply:
that shouldn't be under ``koch`` but w/e